### ISSUE TITLE
Plan community contribution usability test

### ISSUE THREAD

**Issue by henocookie** [nhsuk/nhsuk-service-manual#719]
## What
Plan a usability test for the contribution process.

## Why
To understand:
- What users expect when contributing to the NHS digital service manual
- How well the current contribution process works for users
- Any blockers in the current contribution process

## Relates to
EPIC: Community contribution #685

## Who needs to know about this
Users - for recruitment purposes.
Team - for designing purposes.

## Done when
Tasks we need to do:
- [x] Write the usability test script
- [x] Plan user scenario(s)
- [x] Recruit users
- [x] Plan test sessions including note-takers and observers

## Next steps
- Analyse findings
- Document
- Present
Once results are presented back, we can feed them into the design process.

**Comment by amyj2110** [comment]
# Discussion guide
## Introduction (2 mins)
Thank you so much for your time today.

We're looking at the process for contributing to the NHS digital service manual today. 

- The session will take about 30 to 45 minutes.
- You may have noticed we are joined by [OBSERVERS & NOTE-TAKERS] who are here to take notes for this session. These notes will only be shared with our team and we won’t use your name. Are you okay with this?
- Are you okay if we record the session?
- You can skip any questions or stop at any time.
- There are no right or wrong answers, we're not testing you, your knowledge of the service manual, or your knowledge of the contribution process
- I will refer back to a guidance sheet so I stay on track, it isn't because I’m not listening to you!

I’ll start with some general questions. Then I'll ask you specific questions about the service manual and contribution.

Do you have any questions before we start?

## General questions (2 mins)

- Can you tell me which organisation you work at?
- Can you tell me which team you work on and your role in the team?
- What are you currently working on?
- Service manual usage questions (5 mins)
- Have you used the NHS digital service manual?

If yes
- Can you tell me what you’ve used the service manual for?
- How has using the service manual affected your ways of working?

If no
- Why have you not used the service manual?

- Have you contributed to the NHS digital service manual before?

If yes
- What have you contributed?
- Tell me the process you went through to contribute.

If no
- Why haven't you contributed?
- If you did want to contribute, how would you start the process?

## Contribution process
Ask participant to share screen. Ask participant to open a new tab and begin on blank Google page or whatever their homepage happens to be.

### SCENARIO 1
> You have been prompted by one of the service manual team to share something in the service manual that you recently worked on. Show me how you would go about doing this.

(Ask them to think of a recent piece of work they have been involved with, or present a mock piece of work)

#### HYPOTHESES
- Participants won't know where to go to start a contribution
- Participants won't go to the "Community" section of the service manual to access the GitHub backlog
- Participants won't think to check the backlog before they open a new issue
- Participants won't know what part of their work they should share
- Participants won't know which labels to add to a GitHub issue
- Participants will think to message the person they know best in the service manual team
- Participants will share their work in a message, on a Slack channel
- Participants won't know what will happen after they share something they worked on

#### QUESTIONS
- How long does it take to decide what to do next?
- Where do they go to first?
- Do they go to the backlog on the service manual website?
- Do they go to the service manual backlog on GitHub and if so, do they view the Issues page or Backlog of styles, components and patterns project page?
- Do they send a message to someone in the service manual team?
- Do they post a Slack message in a channel?
- Do they look for a page in the service manual for the type of work they want to share?
- Do they look for an issue in the backlog for the type of work they want to share?
- Do they read other people's contributions?
- What do they share (screenshots, prototype links, user research findings, live URLs etc.)?
- What do they think happens once they're finished?

### SCENARIO 2
> You have noticed a mistake in the service manual. Show me how you would go about rectifying this yourself. (Propose a content change to a service manual page)

#### HYPOTHESES
- Participants will think to message the person they know best in the service manual team
- Participants will know to click something in the "Help improve this page" section on a service manual page
- Participants won't go to the "Community" section of the service manual to access GitHub
- Participants who are not content designers will not know how to make the change
- Participants will know there is some code to change to rectify a mistake
- Participants who haven't used GitHub before won't be confident enough to change code

#### QUESTIONS
- Do they think about messaging a member of the service manual team?
- Do they click on anything in the "Help improve this page" section?
- Do they go to the "Community" section in the service manual?
- Do they know how to change it? Do they know where to make the change?
- Do they know how to nominate someone to approve the PR?
- What do they think will happen once they're finished?
- What do they think will happen if it is approved? What do they think will happen if it is rejected or changes are suggested?

### SCENARIO 3
> Some user research you were part of recently showed that an experimental component wasn't really fit for use. Show me how you would go about sharing your evidence on the service manual. (Share a research or feedback about an existing component or pattern or style guide issue)

#### HYPOTHESES
- Participants won't go to the "Community" section of the service manual to access GitHub
- Participants won't know how to share evidence
- Participants won't know exactly what research to share
- Participants will understand they have to go to a specific page for that component
- Participants will not know to tag someone in their comment
- Participants won't know what happens after they have made the comment

#### QUESTIONS
- What format do they think they would need to share evidence or research?
- Where would they start? Do they know they can go straight to GitHub?
- Do they understand they can leave a comment?
- Do they tag anyone in the comment?
- What do they think will happen once they're finished?

## Thank you & close (2 mins)
Thank you for your time today. It's been really helpful speaking with you. Do you have any questions for me?

Do you have any comments about how the session has gone?

We might ask you to take part in another session - is that ok?

**Comment by amyj2110** [comment]
Summary of results can be found in #825 - Write up community contribution usability testing.
